Weather.СonsoleApp/Weather.СonsoleApp/Program.cs

```diff
@@ -24,47 +26,37 @@ static async Task Main(string[] args)
 
             _weatherService = kernel.Get<IWeatherService>();
 
-            bool showMenu = true;
-            while (showMenu)
-            {
-                showMenu = await MainMenu();
-            }
-        }
+            ICommand getWeather = new GetWeatherCommand(_weatherService);
+            ICommand getForecast = new GetForecastCommand(_weatherService);
 
-        private static async Task<bool> MainMenu()
-        {
-            Console.WriteLine("Select an action:");
-            Console.WriteLine("If you want to know the weather in the city, enter 1");
-            Console.WriteLine("If you want to close the application, enter 2");
+            var listCommand = new List<ICommand>();
```

Comment:
var listCommand = new List<ICommand>{ getWeather, getForecast };
I think it will be better 🙂